bodhi/client/__init__.py

```diff
@@ -587,11 +592,11 @@ def query(url, debug, mine=False, rows=None, **kwargs):
     Query updates based on flags.
 
     Args:
-        url (unicode): The URL of a Bodhi server to create the update on. Ignored if staging is
+        url: The URL of a Bodhi server to create the update on. Ignored if staging is
                        True.
-        mine (Boolean): If the --mine flag was set
-        debug (Boolean): If the --debug flag was set
-        kwargs (dict): Other keyword arguments passed to us by click.
+        mine: If the --mine flag was set
+        debug: If the --debug flag was set
```

Comment:
`rows` is missing + what do you think about changing order of args in `Args` section to match the order of args in function?

Reply:
I'll do a third commit to switch the arg order to match.